setup/src/installer.ts

```diff
@@ -94,8 +95,8 @@ async function isInstalled(
   };
 
   for (const p of locations[tool]) {
-    const installedPath = await fs
-      .access(p)
+    const installedPath = await afs
+      .access(p || 'undefined')
```

Comment:
Why was this was changed to `|| 'undefined'`? Is there an error condition I was missing earlier?

Reply:
`locations['cabal']['win32']` was being populated with non-sensical paths on platforms other than Windows, which confused me.

I've removed the changes.